elasticdl/python/tests/pserver_servicer_test.py

```diff
@@ -259,18 +259,18 @@ def push_gradient_test_setup(self):
         self.embedding_table = (
             np.random.rand(4 * dim).reshape((4, dim)).astype(np.float32)
         )
-        self.embedding_grads = tf.IndexedSlices(
+        self.embedding_grads0 = tf.IndexedSlices(
             values=np.random.rand(3 * dim)
             .reshape((3, dim))
             .astype(np.float32),
             indices=(3, 1, 3),
         )
-        self.expected_embed_table = np.copy(self.embedding_table)
-        for gv, gi in zip(
-            self.embedding_grads.values, self.embedding_grads.indices
-        ):
-            self.expected_embed_table[gi] -= self._lr * gv
-
+        self.embedding_grads1 = tf.IndexedSlices(
```

Comment:
Just to address the issue [here](https://github.com/sql-machine-learning/elasticdl/pull/1439#discussion_r344990022), refine the unit test.